File: test/net/dinkla/raytracer/samplers/RegularTest.java
package net.dinkla.raytracer.samplers;

import org.junit.jupiter.api.BeforeEach;

public class RegularTest extends GeneratorTest {

    @BeforeEach
    @Override
    public void initialize() {
        new Regular().generateSamples(NUM_SAMPLES, NUM_SETS , samples);
    }
    
}

File: src/test/kotlin/net/dinkla/raytracer/samplers/RegularTest.kt
package net.dinkla.raytracer.samplers

import org.junit.jupiter.api.BeforeEach

class RegularTest : AbstractGeneratorTest() {

    @BeforeEach
    override fun initialize() {
        Regular().generateSamples(NUM_SAMPLES, NUM_SETS, samples)
    }

}
